tests/console/commands/debug/test_resolve.py

```diff
@@ -9,14 +9,20 @@ def tester(command_tester_factory):
     return command_tester_factory("debug resolve")
 
 
-def test_debug_resolve_gives_resolution_results(tester, repo):
+@pytest.fixture(autouse=True)
+def __add_packages(repo):
     cachy2 = get_package("cachy", "0.2.0")
     cachy2.add_dependency(Factory.create_dependency("msgpack-python", ">=0.5 <0.6"))
```

Comment:
```suggestion
    cachy020 = get_package("cachy", "0.2.0")
    cachy020.add_dependency(Factory.create_dependency("msgpack-python", ">=0.5 <0.6"))
```

While we are doing this, might has well name this right. :)

Reply:
Done :)